internal/slacklog/message.go

```diff
@@ -363,6 +363,12 @@ type MessageFile struct {
 	HasRichPreview     bool   `json:"has_rich_preview"`
 }
 
+func (f *MessageFile) IsHostedFile() bool {
```

Comment:
名前を変えてdocコメントつけましょう。
どこにホストされてるか明かにするのと、Fileは自明なので省略するのと

```suggestion
// IsSlackHosted returns true when this file is hosted by Slack
func (f *MessageFile) IsSlackHosted() bool {
```

Reply:
あ、日本語で書いちゃった…。

Reply:
まぁええよw